test/scss/feature-targeting.scss

```diff
@@ -24,19 +24,25 @@
   @include mdc-button-shape-radius(0, $query: mdc-feature-any());
   @include mdc-button-horizontal-padding(0, $query: mdc-feature-any());
   @include mdc-button-outline-width(0, $query: mdc-feature-any());
+  @include mdc-button-ripple($query: mdc-feature-any());
```

Comment:
IIUC these don't need to be here because `*-core-styles` already exercise them indirectly. Do you have a strong opinion on them being here for consistency's sake?

Reply:
I've been adding all public mixins, that way we don't have to think about which ones are included by other public mixins already. It doesn't hurt since it should all boil down to no styles emitted in the end